integration-tests/azure-cosmos/src/main/java/io/quarkiverse/azure/cosmos/it/CosmosAsyncResource.java

```diff
@@ -0,0 +1,101 @@
+package io.quarkiverse.azure.cosmos.it;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.UriInfo;
+
+import com.azure.cosmos.*;
+import com.azure.cosmos.models.*;
+
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
```

Comment:
At first I was concerned about this dependency, but now I see it's all over the place in Azure SDK for Java, so I guess it's OK.